pom.xml

```diff
@@ -104,6 +103,18 @@
       <artifactId>maven-resolver-api</artifactId>
       <version>1.6.3</version>
     </dependency>
+    <dependency>
+      <groupId>net.jcip</groupId>
+      <artifactId>jcip-annotations</artifactId>
+      <version>1.0</version>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>com.github.spotbugs</groupId>
+      <artifactId>spotbugs-annotations</artifactId>
+      <version>4.7.1</version>
```

Comment:
Version is already managed in the parent POM. Since this is the case, omitting it here will reduce Dependabot spam.

```suggestion
```